test/controllers/health_cards_controller_test.rb

```diff
@@ -45,7 +45,10 @@ class HealthCardsControllerTest < ActionDispatch::IntegrationTest
     get(chunks_patient_health_card_url(@patient))
     assert_response :success
 
-    json = JSON.parse(response.body)
-    assert_equal 1, json.size
+    chunks = JSON.parse(response.body)
+    # Check that each string in the array is a numeric string
+    chunks.each do |s|
+      assert_empty(s.scan(/\D/))
```

Comment:
Can we do any other assertions based on the contents aside from numeric?

Reply:
Yep, I've updated the test to reconstruct the jws and check that the bundle entries match.